srcpkgs/i3/template

```diff
@@ -14,7 +14,7 @@ maintainer="Orphaned <orphan@voidlinux.org>"
 license="BSD-3-Clause"
 homepage="https://i3wm.org/"
 distfiles="${homepage}/downloads/${pkgname}-${version}.tar.xz"
```

Comment:
```suggestion
distfiles="https://i3wm.org/downloads/i3-${version}.tar.xz"
```

Reply:
hi, can I know why use the literals instead of variables? just curious (as still in early stages on contributing to void)

Reply:
$pkgname isn't really variable, it can easily change when the package is renamed and $homepage is annoying because people can not easily get to the downloads page by just clicking on the link and it gets in the way of grepping through templates without evaluating them. It's not wrong to use the variables, many maintainers just prefer to not use them.

Reply:
cool, makes sense, will update this PR asap, thanks!